cobigen/cobigen-core-api/src/test/java/com/devonfw/cobigen/api/MavenUtilTest.java

```diff
@@ -1,330 +0,0 @@
-package com.devonfw.cobigen.api;
-
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
-import static com.github.tomakehurst.wiremock.client.WireMock.get;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
-import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
-import static org.assertj.core.api.Assertions.assertThat;
-
-import java.io.IOException;
-import java.net.URL;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.util.List;
-
-import org.junit.Rule;
-import org.junit.Test;
-
-import com.devonfw.cobigen.api.exception.RestSearchResponseException;
-import com.devonfw.cobigen.api.util.MavenUtil;
-import com.devonfw.cobigen.api.util.to.AbstractSearchResponse;
-import com.devonfw.cobigen.api.util.to.jfrog.JfrogSearchResponse;
-import com.devonfw.cobigen.api.util.to.maven.MavenSearchResponse;
-import com.devonfw.cobigen.api.util.to.nexus2.Nexus2SearchResponse;
-import com.devonfw.cobigen.api.util.to.nexus3.Nexus3SearchResponse;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.github.tomakehurst.wiremock.junit.WireMockRule;
-
-/**
- * Test class for maven utilities
- */
-public class MavenUtilTest {
-
-  /**
-   * WireMock rule to initialize
-   */
-  @Rule
-  public WireMockRule wireMockRule = new WireMockRule(options().disableRequestJournal());
-
-  /** Testdata root path */
-  private static final String testdataRoot = "src/test/resources/testdata/unittest/MavenUtilTest";
-
-  /**
-   * Tests if retrieving maven artifacts with an invalid link returns null
-   */
-  public void testRetrieveMavenArtifactsWithInvalidLinkReturnsNull() {
-
-    assertThat(MavenUtil.retrieveMavenArtifactsByGroupId("this/is/not/a/link", "test", null)).isNull();
-  }
-
-  /**
-   * Tests if a {@link RestSearchResponseException} gets thrown when a faulty target link without a token was used
-   */
-  @Test
-  public void testWrongTargetLinkThrowsException() {
-
-    try {
-      AbstractSearchResponse.retrieveJsonResponseWithAuthenticationToken("this/is/not/a/link", null, null);
-    } catch (RestSearchResponseException e) {
-      assertThat(e).hasMessage("The target URL was faulty.");
-    }
```

Comment:
This is not a reliable way to test that an Exception is thrown, the test won't fail if the exception isn't thrown. Use: assertThatExceptionOfType(RestSearchResponseException.class).isThrownBy(() -> 'your code').withMessage("expected message").